assets/js/settings.js

```diff
@@ -71,6 +77,14 @@ jQuery( document ).ready( function ( $ ) {
 		return false;
 	} );
 
+	function markSectionAsVisited( sectionId ) {
+		const data = new FormData();
+		data.append( 'action', 'sensei_settings_section_visited' );
+		data.append( 'sectionId', sectionId );
```

Comment:
Would it make sense to transform the value that we'll use in PHP to snake case? And maybe convert the `sensei_settings_section_visit_nonce` to be camel case since we just use it in JS?

Reply:
Done in 0fa8ec82e